packages/clerk-js/src/core/resources/PublicUserData.ts

```diff
@@ -1,8 +1,8 @@
 import { deprecatedProperty } from '@clerk/shared';
-import type { PublicUserData } from '@clerk/types';
+import type { PublicUserData as IPublicUserData } from '@clerk/types';
 import type { PublicUserDataJSON } from '@clerk/types';
 
-export class OrganizationPublicUserData implements PublicUserData {
+export class PublicUserData implements IPublicUserData {
```

Comment:
❓ Why are we renaming this class?

Reply:
Because the `publicUserData` property is the same for all Resources (eg `OrganizationMembership `, `Session`), so there is no need to create a separate class for each resource.